source/getting-started/use_cases/configuration_assessment.rst

```diff
@@ -71,8 +71,25 @@ macOS SCA rule example:
 
 Below is an example of the results of a configuration assessment evaluation. These can be obtained through the web user interface or directly through the Wazuh RESTful API.
 
-.. thumbnail:: ../../images/getting_started/use_case_sca.png
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-sca1.png
```

Comment:
These images need to be edited. I can see the black line from the header you removed. 

![image](https://user-images.githubusercontent.com/79285586/141320938-77b044d5-b707-453c-9887-47ec948cd1ae.png)